util/execdetails/execdetails.go

```diff
@@ -253,11 +253,49 @@ func (crs *CopRuntimeStats) String() string {
 		procTimes[n-1], procTimes[0], procTimes[n*4/5], procTimes[n*19/20], totalRows, totalIters, totalTasks)
 }
 
+// ReaderRuntimeStats collects stats for TableReader, IndexReader and IndexLookupReader
+type ReaderRuntimeStats struct {
+	sync.Mutex
+
+	copRespTime []time.Duration
+}
+
+// recordOneCopTask record once cop response time to update maxcopRespTime
+func (rrs *ReaderRuntimeStats) recordOneCopTask(t time.Duration) {
+	rrs.Lock()
+	defer rrs.Unlock()
+	rrs.copRespTime = append(rrs.copRespTime, t)
+}
+
+func (rrs *ReaderRuntimeStats) String() string {
+	sort.Slice(rrs.copRespTime, func(i, j int) bool {
+		return rrs.copRespTime[i] < rrs.copRespTime[j]
+	})
+	s := "rpc time "
+	size := len(rrs.copRespTime)
+	if size == 0 {
+		return s + "max:0ns, min:0ns, avg:0ns, p80:0ns, p95:0ns"
```

Comment:
we can just return an empty string if there is no cop task.

Reply:
Yes, but the cop runtime stats do not distinguish either.